config/deploy/production.rb

```diff
@@ -0,0 +1,6 @@
+user = 'raymoond'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.